internal/services/appservice/function_app_hybrid_connection_resource.go

```diff
@@ -231,7 +231,7 @@ func (r FunctionAppHybridConnectionResource) Read() sdk.ResourceFunc {
 				}
 
 				hybridConnectionsClient := metadata.Client.Relay.HybridConnectionsClient
-				hybridConnectionID, err := hybridconnections.ParseHybridConnectionID(appHybridConn.RelayId)
+				hybridConnectionID, err := hybridconnections.ParseHybridConnectionIDInsensitively(appHybridConn.RelayId)
```

Comment:
we're parsing the same value twice here (and above).. is this correct?

Reply:
looks duplicated, I've removed the second parse